### ISSUE TITLE
Add mechanics and items from Pokémon legends areceus (!!Spoilers for the switch game are in here)

### ISSUE THREAD

**Issue by Nathan22211** [Pokecube-Development/Pokecube-Issues-and-Wiki#895]
**Is your feature request related to a problem? Please describe.**
Not related to a problem, it's just that now you're missing some stuff from the most recent game

**Describe the solution you'd like**
Items and mechanics from the game added to the mod. Check the context as I have a few ideas for that

**Describe alternatives you've considered**
This is fairly recent so I doubt anyone else is doing an addon for this. Besides dodging (I think elini dodge is in 1.16)

**Additional context**
as far as the old-style pokeballs they should be easier to craft but has a lesser chance of working than their modern counterparts. I can probably tweet out screenshots of the recipes of them in the switch game but I don't have all of them unlocked.

There's now agile and strong style moves, but they need to master the moves which seem to happen after a few levels. I don't know how you'll integrate this into real-time combat though.

As far as the pokemobs, so far they're all hostile if you get too close. But now some pokemon, e.g. bidoof, won't go after the player and others will run away. At times I've noticed they'll attack if you try to catch them, but not with all of them. Some pokedex entries even say that some will approach people.

The alpha and frenzied pokemon I'm a bit strapped for ideas there. Same for the arc phone, and I haven't gotten all of the plates yet in the game so that's all I can give for now.

**Comment by xigzagoon** [comment]
perhaps agile moves could be weaker but have less cooldown while strong moves could be stronger but have more cooldown
The ability to add back "forgotten" moves at-will as pokemon can in Legends Arceus would also be cool
Pokemon could have a small chance of spawning as Alphas, which would be larger, have glowing eyes, be able to pull a few moves from a different movepool, and be higher levels than the surrounding pokemon as well as being very aggressive.

It may be prudent to wait and see if the "mainline" games start using frostbite and drowzy, and only looking into implementing those over freeze and sleep if they do.

Being able to throw Feather Balls more like snowballs and them catching what they strike would be pretty cool :D

**Comment by Nathan22211** [comment]
you can chuck pokecubes pretty far as is. I've chucked mine maybe a few chunks away when trying to catch some on the server

**Comment by Nathan22211** [comment]
yeah just played scarlet and they reintroduced sleeping, but not sure about freezing as I almost never encounter that
